packages/react-component-library/src/components/Autocomplete/AutocompleteOption.tsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
 import reactStringReplace from 'react-string-replace'
 
-import { SelectBaseOption, SelectBaseOptionAsStringProps } from '../SelectBase'
+import { SelectBaseOption, SelectBaseOptionProps } from '../SelectBase'
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface AutocompleteOptionProps extends SelectBaseOptionAsStringProps {
+export interface AutocompleteOptionProps extends SelectBaseOptionProps {
```

Comment:
Is there any need to change the external type here? (Assume it was restricted to a string for children on purpose.)